docs/source/admin/quick_howto/static_dns.rst

```diff
@@ -0,0 +1,77 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _static-dns-qht:
+
+******************************
+Configuring Static DNS Entries
+******************************
+Any number static DNS records (historically "entries") can be configured on a per-:term:`Delivery Service` basis. In a typical scenario, the :term:`Delivery Service` will have DNS records automatically generated based on its "xml_id" and "routing name", and the name and subdomain of the CDN to which it belongs. For example, in the :ref:`ciab` default environment, the "demo1" :term:`Delivery Service` has an automatically generated DNS record for ``video.demo1.mycdn.ciab.test``. Configuring a static DNS record allows for further extension of this, for example, one could create an ``A`` record that enforces lookups of the name ``foo.demo1.mycdn.ciab.test`` resolve to the IPv4 address ``192.0.2.0``.
```

Comment:
The `(historically "entries")` bit seems superfluous. I'd also remove the `Any number` part and add `within the Delivery Service's subdomain` to the end of the first sentence.

Also, `*.0` addresses are typically reserved as the "network address", so a better example would probably be something between .1 and .254 (.255 is typically reserved as the "broadcast address").

Reply:
¯\\\_(ツ)\_/¯ if you say so.

I'd like to leave in the `(historically "entries")` bit because otherwise it could be confusing that the reader is in the "Configuring Static DNS **Entries**" section but it just keeps talking about **records**. I think the connection is important to make - after all _I_ didn't know they were the same thing so it seems unfair of me to expect others to know.